Assets/LeapMotionModules/GraphicRenderer/Scripts/LeapGraphic.cs

```diff
@@ -8,7 +8,7 @@ namespace Leap.Unity.GraphicalRenderer {
 
   [ExecuteInEditMode]
   [DisallowMultipleComponent]
-  public abstract partial class LeapGraphic : MonoBehaviour, ISpaceComponent {
+  public abstract partial class LeapGraphic : MonoBehaviour, ISpaceComponent, IEquatable<LeapGraphic> {
```

Comment:
Did you experience a noticeable difference due to the overhead reduction implementing IEquatable provides when used in HashSets and Dictionaries? If so I should probably make InteractionBehaviours and any other particularly important components also implement IEquatable...